tutorials/getting_started/getting_started.md

```diff
@@ -0,0 +1,181 @@
+
+If you can't (or don't want to) use Docker, you can clone from our git repository:
+
+```bash
+git clone https://github.com/allenai/allennlp.git
+```
+
+Create a Python 3.5 (or 3.6) virtual environment, and run
+
+```bash
+INSTALL_TEST_REQUIREMENTS=true scripts/install_requirements.sh
+```
+
+changing the flag to `false` if you don't want to be able to run tests.
+(Narrator: You want to be able to run tests.)
+
+You'll also need to install PyTorch 0.2, following the appropriate instructions
+from [their website](http://pytorch.org/).
+
+## Once You've Installed
+
+A lot of the most common functionality can be accessed through the command line tool `allennlp/run`:
+
+```
+(allennlp) root@9175b60b4e52:/stage# allennlp/run
+usage: run [command]
+
+Run AllenNLP
+
+optional arguments:
+  -h, --help  show this help message and exit
+
+Commands:
+
+    bulk      Run a model in bulk.
+    train     Train a model
+    serve     Run the web service and demo.
+    evaluate  Evaluate the specified model + dataset
+```
+
+### Serving the Demo
+
+The `serve` command starts the demo server.
+
+```
+(allennlp) root@9175b60b4e52:/stage# allennlp/run serve
+Starting a flask server on port 8000.
+[... lots of logging omitted ...]
+2017-08-14 20:00:48,330 - INFO - werkzeug -  * Running on http://0.0.0.0:8000/ (Press CTRL+C to quit)
+```
+
+If you visit `http://localhost:8000` in your browser, you can play around with the same demo
+that runs on the AllenNLP website.
+
+TODO(joelgrus): screenshot
+
+### Training a Model
+
+In this tutorial we'll train a simple part-of-speech tagger using AllenNLP.
+The model is defined in [allennlp/models/simple_tagger.py](https://github.com/allenai/allennlp/blob/master/allennlp/models/simple_tagger.py).
+It consists of a word embedding layer followed by a LSTM.
+
+Our dataset will be a subset of the [Brown Corpus](http://www.nltk.org/nltk_data/).
+In particular, the file [tutorials/getting_started/data/cr.train](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/data/cr.train)
+is the concatenation of the Brown files `cr01`, ..., `cr08` (that's the "humor" category)
+and [tutorials/getting_started/data/cr.dev](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/data/cr.test) is the file `cr09`.
+
+In AllenNLP we configure experiments using JSON files. Our experiment is defined in
+[tutorials/getting_started/simple_tagger.json](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/simple_tagger.json). You can peek at it
+if you want, we'll go through it in detail in the next tutorial.  Right at this instant
+you might care about the `trainer` section, which specifies how we want to train our model:
+
+```js
+  "trainer": {
+    "num_epochs": 20,
+    "serialization_prefix": "/tmp/tutorials/getting_started",
+    "cuda_device": -1
+  }
+```
+
+The `num_epochs` parameter just specifies how many passes through the dataset we want to train on.
```

Comment:
I don't want to say too much, I think our users know what an epoch is, but I reworded it a bit